src/featuremap/tests/unit/test_featuremap_xgb_prediction.py

```diff
@@ -19,11 +19,11 @@ def test_pileup_featuremap(
     tmpdir,
     resources_dir,
 ):
-    input_featuremap_vcf = pjoin(resources_dir, "featuremap_vcf_for_xgb_prediction.vcf.gz")
+    input_featuremap_vcf = pjoin(resources_dir, "five_giab.ppmSeq.dup_filtered.pileup.chr9.vcf.gz")
```

Comment:
I think we need two tests: one for the ppmSeq model and another for the standard model